Andre-Dinh/index.html.html

```diff
@@ -0,0 +1,145 @@
+          </section>
+          <section>
+            <h2>SUSPENSE</h2>
+            <h3> Future packages will be processed the 15th of every month and shipped shortly thereafter</h3>
+          </section>
+        <button> GET STARTED</button>
+      </section>
+      <section>
+        <video>Vimo</video>
+      </section>
+      <section>
+      <section>
+        <h2>WHO WE ARE</h2>
+        <p>Five Four is the brand. Five Four Club is the service and the community that we support.</p>
+      </section>
+      <section>
+        <h2>Convenience</h2>
+        <ol>
+          <li> Monthly delivery to preserve your free time.</li>
+          <li>Offering "how to wear' advice to remove guesswork.</li>
+        </ol>
+      </section>
+      <section>
+        <h2>Confidence</h2>
+        <ol>
+          <li>Striving for more without fearing.</li>
+          <li>Receive a 25% discount in our online store.</li>
+        </ol>
+      </section>
+      <section>
+        <h2>Acessibility</h2>
+        <ol>
+          <li>An inclusive brotherhood embracing diversity.</li>
+          <li>supporting the unique and varied pursuits of life.</li>
+        </ol>
+      </section>
+      <section>
+        <h2>Style</h2>
+        <ol>
+          <li>contemporary designs exlusive to Members</li>
+          <li>An independent fashion brand that is one house, not a ware house.</li>
+        </ol>
+      </section>
+      <button>GET STARTED</button>
+    </section>
+    <section>
+      <h2>HASSSLE-FREE SHIPPING</h2>
+      <p>Your package is delivered right to your doorsettop every month. Shipping is on us*.</p>
+    </section>
+    <section>
+      <h2>FREE SHIPPING</h2>
+      <p>shipping is free both ways.*</p>
+    </section>
+    <section>
+      <h2>EASY SIZE EXCHANGES</h2>
+      <p>A prepaid label is provided for size exchages. Email us and we'll walk you through the process.</p>
+    </section>
+    <section>
+      <h2> SHIPPING</h2>
+      <p>We currently service all 50 States, Canada,& UK. We're also working on expanding our reach.</p>
+    </section>
+    <button>GET STARTED</button>
+    <section>
+      <h2>WHAT YOU RECEIVE</h2>
+      <p>Each month you'll receive a different package of clothing according yo your style profile and preferences.</p>
+    </section>
+    <section>Types of clothing
+      <h2>type of clothing</h2>
+      <p>Discription of clothing</p>
+      <a href=""></a>
+      <p>price of each clothing and savings</p>
+    </section>
+    <button>GET STARTED</button>
+    <section>Featured In
+      <p>times, attack of the show, pandodaily,  g4, techCrunch, mashable, wsj, los angeles times
+      </p>
+    </section>
+      <section>WE'LL MAKE YOU LOOK GREAT. TRUST US.
+    </section>
+    <section>
+    <footer>
+      <section>*AK/CA/HI/Canadian/United Kingdom members will be charged applicable tax and/or additional shipping fees.</section>
+      <section>facebook like</section>
+      <nav>
+        <ol>
```

Comment:
Try not to use UPPERCASE text in your markup. It's presentational and should be handled by CSS